### PR TITLE
Add New Featured Artist: WINTERHORDE news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
Some might say this artist gives people the chills.

### CODE REVIEW

File: news/2023/2023-01-25-new-featured-artist-winterhorde.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/337/The%20Greatest%20Plague%20of%20Earth%2FWinterhorde%20-%20The%20Greatest%20Plague%20of%20Earth.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+[**WINTERHORDE**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/337) has even more songs for you to smash your keyboard along to. Pick whichever suits your fancy and design your own finger-demolishing canvas of chaos (also known as a beatmap).
+
+Our next Featured Artist might be a day late for... undisclosed reasons. Keep an eye on the [@osugame Twitter](https://twitter.com/osugame) for when it drops, and maybe watch this weekend's [MWC7K](/wiki/Tournaments/MWC/2023_7K) for... similar undisclosed reasons.
```
Comment (Nivalyx) [inline news/2023/2023-01-25-new-featured-artist-winterhorde.md:49]:
```suggestion
Our next Featured Artist might be a day late for... undisclosed reasons. Keep an eye on the [@osugame Twitter](https://twitter.com/osugame) for when it drops, and maybe watch this weekend's [MWC 7K](/wiki/Tournaments/MWC/2023_7K) for... similar undisclosed reasons.
```

Comment (Nivalyx) [reply news/2023/2023-01-25-new-featured-artist-winterhorde.md:49]: (it's spelled this way according to the wiki)

image

File: news/2023/2023-01-25-new-featured-artist-winterhorde.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "New Featured Artist: WINTERHORDE"
+date: 2023-01-25 11:00:00 +0000
+---
+
+Melodic black/death metal band **WINTERHORDE** takes a step into our library of Featured Artist.
```
Comment (Walavouchey) [inline news/2023/2023-01-25-new-featured-artist-winterhorde.md:7]:
```suggestion
Melodic black/death metal band **WINTERHORDE** takes a step into our library of Featured Artists.
```

```suggestion
Melodic black/death metal band **WINTERHORDE** takes a step into our library as a Featured Artist.
```

File: news/2023/2023-01-25-new-featured-artist-winterhorde.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+![](https://assets.ppy.sh/artists/337/header.jpg)
+
+**WINTERHORDE**'s music is everything a metalhead could ask for: high-tempo, high-intensity, and abundantly dense with drums. Stream-focused metal mapping (which should really be called *blast beatmapping*) is their speciality, as made crystal clear by maps like [Chronic Death](https://osu.ppy.sh/beatmapsets/1699956) and [Worms of Soul](https://osu.ppy.sh/beatmapsets/1724057).
+
+Wrench your mind around the **6** tracks we've picked up for [**WINTERHORDE**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/337), all pre-timed and ready for your Ctrl+Shift+F mastery.
```
Comment (Walavouchey) [inline news/2023/2023-01-25-new-featured-artist-winterhorde.md:13]:
```suggestion
Wrench your mind around the **6** tracks we've picked up for [**WINTERHORDE**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/337), all pre-timed and ready for your `Ctrl` + `Shift` + `F` mastery.
```

File: news/2023/2023-01-25-new-featured-artist-winterhorde.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Check out [the map from the video above](https://osu.ppy.sh/beatmapsets/1724057) hosted by [kiwies](https://osu.ppy.sh/users/12927633)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/337/Maestro/Winterhorde%20-%20Worms%20of%20Soul.mp3" type="audio/mpeg">
+</audio>
+
+### WINTERHORDE - Chronic Death
+
+Or try [this other ultra-difficult marathon](https://osu.ppy.sh/beatmapsets/1699956) hosted by [iamtickrate](https://osu.ppy.sh/users/10772580)!
```
Comment (Walavouchey) [inline news/2023/2023-01-25-new-featured-artist-winterhorde.md:33]:
```suggestion
Or try [this other ultra difficult marathon](https://osu.ppy.sh/beatmapsets/1699956) hosted by [iamtickrate](https://osu.ppy.sh/users/10772580)!
```

**Review by Walavouchey — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**